packages/api/src/services/actions/global.ts

```diff
@@ -1,71 +1,150 @@
 import assert from "assert";
 import { Json, Actions, AppState, CurrencySymbol, AllContractStates } from "../..";
-import * as Queries from "../../libs/queries";
 import bluebird from "bluebird";
 import { BigNumber } from "ethers";
 import { nowS } from "../../libs/utils";
+import { Handlers as EmpActions } from "./emp";
+import { Handlers as LspActions } from "./lsp";
```

Comment:
im using the other action channels within the global channel so i can reuse the logic